crates/claims/crates/status/src/client/http.rs

```diff
@@ -0,0 +1,60 @@
+use iref::Uri;
+
+use crate::{EncodedStatusMap, FromBytes, FromBytesOptions};
+
+use super::{MaybeCached, ProviderError, StatusMapProvider, TypedStatusMapProvider};
+
+pub struct HttpClient<V> {
+    client: reqwest::Client,
```

Comment:
I'm fine with using `reqwest` directly for the time being, but in the future we should use a trait similar to what `openidconnect` does https://docs.rs/openidconnect/4.0.0-alpha.2/openidconnect/trait.AsyncHttpClient.html